client/hosting/performance/components/PageSelector.tsx

```diff
@@ -16,12 +16,24 @@ export const PageSelector = ( props: ComponentProps< typeof SearchableDropdown >
 				<SearchableDropdown
 					{ ...props }
 					className="site-performance__page-selector-drowdown"
-					__experimentalRenderItem={ ( { item } ) => (
-						<div className="site-performance__page-selector-item" aria-label={ item.label }>
-							<span>{ item.label }</span>
-							<span className="subtitle">{ item.path }</span>
-						</div>
-					) }
+					__experimentalRenderItem={ ( { item } ) => {
+						if ( item.value === '-1' ) {
+							return (
+								<div className="message">
+									{ translate( 'Searching your top 20 most popular pages.' ) }
+								</div>
+							);
+						}
+						if ( item.value === '-2' ) {
+							return <div className="message">{ translate( 'No pages found' ) }</div>;
```

Comment:
Since we're looking for `a` specific page when using the searching, should it be `Page not found` instead?

Reply:
I think because a search query can return a list of pages matching and not only a specific one `No pages found` works. I think the copy works as is. but we can change later if others have a strong opinion.